app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) -> any:
```

Comment:
`__init__` doesn't return anything and should always return `None`. The same in other similar cases